packages/orbit-components/src/Tag/index.js.flow

```diff
@@ -14,6 +14,7 @@ export type Props = {|
   +icon?: React.Node,
   +selected?: boolean,
   +size?: Size,
+  +ref?: Ref,
```

Comment:
Instead of this it's better to export a `React.AbstractComponent`, see `InputField/index.js.flow` as an example. There's no need for the `Ref` utility at all.

Reply:
it's bad example :D there is spread Ref type, we should fix that